data/mp/multiplay/skirmish/Cobra.json

```diff
@@ -0,0 +1,7 @@
+{
+    "AI": {
+        "js": "Cobra.js",
+        "name": "Cobra",
+        "tip": "Designed for team battles."
+    }
```

Comment:
Could descriptions for different difficulty levels be added here?

Reply:
Yep. Will do.